files/verify-cn.yml

```diff
@@ -1,3 +1,4 @@
 ---
-realm: COOL.CYBER.DHS.GOV
+log_level: INFO
+realm: STAGING.COOL.CYBER.DHS.GOV
```

Comment:
Is this `STAGING.` supposed to be here?

Reply:
yes. This file is just a placeholder.  The real file is created by cloud-init.